apps/native/app/src/utils/get-main-root.ts

```diff
@@ -10,41 +10,61 @@ import {
 import { getThemeWithPreferences } from './get-theme-with-preferences'
 import { testIDs } from './test-ids'
 
+type Screen = 'Inbox' | 'Wallet' | 'Home' | 'Applications' | 'More'
+
 type RightButtonProps = {
   unseenCount?: number
   theme?: ReturnType<typeof getThemeWithPreferences>
+  screen?: Screen
 }
 
 export const getRightButtons = ({
   unseenCount = notificationsStore.getState().unseenCount,
   theme = getThemeWithPreferences(preferencesStore.getState()),
+  screen,
 }: RightButtonProps = {}): OptionsTopBarButton[] => {
   const iconBackground = {
```

Comment:
~return early with empty array if no screens or icons depending if you change it